Cubical/Algebra/LindenbaumTarski.agda

```diff
@@ -0,0 +1,391 @@
+{-# OPTIONS --safe #-}
```

Comment:
Maybe add a short comment in the top of the file saying what it contains?

Reply:
Valid point!

Reply:
Great!